### PR TITLE
[doc] Add a schema of the tokenlist response

### PR DESCRIPTION
**Pull request by dcermak**
This schema has been created by guessing, so it might be wrong and incomplete.

### CODE REVIEW

**Comment by krauselukas** [comment]
@dcermak having a look

**Review by krauselukas — COMMENTED** [review]
@dcermak I'm not an expert with those xml schemas, but it looks correct to me. Don't know if its mandatory, but you didn't provided an xml example. I saw in most other part there is one, maybe you could add one?

**Comment by dcermak** [comment]
Lukas Krause <notifications@github.com> writes:

>
> @dcermak I'm not an expert with those xml schemas, but it looks correct to me. Don't know if its mandatory, but you didn't provided an xml example. I saw in most other part there is one, maybe you could add one?
>

I have added a small one

**Comment by hennevogel** [comment]
The CI fails to validate the schema. See https://app.circleci.com/pipelines/github/openSUSE/open-build-service/2125/workflows/08ceca49-3a06-4234-844c-9d496cdede56/jobs/53767

**Comment by adrianschroeter** [comment]
JFYI, a rng schema would be prefered, but we can also take an xsd (it would always enforce element ordering though).
No need to rewrite it now, but for the future.

**Comment by dcermak** [comment]
> The CI fails to validate the schema. See https://app.circleci.com/pipelines/github/openSUSE/open-build-service/2125/workflows/08ceca49-3a06-4234-844c-9d496cdede56/jobs/53767

I hope it's fixed now :crossed_fingers:

**Comment by dcermak** [comment]
Next try, this time with a rng schema.

File: src/api/test/functional/backend_test.rb [hunk 1]
```diff
@@ -21,7 +21,7 @@ def test_validate_bsxml
           'productlist', 'binary_released', 'check', 'required_checks', 'status_report',
           'staged_requests', 'remove_staged_requests', 'status_ok', 'staging_project',
           'staging_projects', 'create_staging_workflow', 'update_staging_workflow', 'accept_staging_projects',
-          'excluded_requests', 'create_excluded_requests', 'delete_excluded_requests', 'create_staging_projects', 'backlog'].include?(schema)
+          'excluded_requests', 'create_excluded_requests', 'delete_excluded_requests', 'create_staging_projects', 'backlog', 'tokenlist'].include?(schema)
```
Comment (dcermak) [inline src/api/test/functional/backend_test.rb:24]: @mlschroe I am not sure if there is a backend schema for the tokenlist. Do you happen to know that?

**Review by hennevogel — APPROVED** [review]

**Pull request merged**